src/components/organisms/Carousel/Carousel.scss

```diff
@@ -1,148 +1,118 @@
-.carousel {
-  top: 65px;
-  width: 100%;
+@import "../../../App.scss";
+
+.carousel-container {
   position: relative;
-  overflow: hidden;
+  padding-top: 65px;
+  width: 100%;
 }
 
-.carousel__slider { 
-  display: flex;
+.carousel-container__carousel {
   width: 100%;
-  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  overflow: hidden;
 }
 
-.carousel__slider__image-slider {
-  position: relative;
-  width: 100%;
+.carousel-container__carousel__slider {
+  display: flex;
+  width: 300%;
+  flex-shrink: 0;
+  transition: all 0.5s;
 }
 
-.carousel__slider__image-slider img{
-  position: absolute;
-  width: 100%;
-  height: auto;
+.carousel-container__carousel__slider section {
+  width: calc(100% / 3);
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
-.carousel__slider__text-slider {
-  margin-top: 60%;
-  position: absolute;
-  color: white;
+
+.carousel-container__carousel__slider section img {
   width: 100%;
-  height: 20%;
 }
 
-.text-block {
-  padding-right: 20px;
-  display: block;
+.carousel-text-container {
+  top: 53%;
   width: 100%;
+  position: absolute;
 }
 
-.carousel__slider__text-slider .heading {
-  top: 20px;
-  left: 18px;
-  white-space: nowrap;
+.carousel-text-container__text-carousel {
+  width: 100%;
+  height: 150px;
+  display: flex;
+  justify-content: flex-start;
+  overflow: hidden;
   position: absolute;
-  font-size: 28px;
-  letter-spacing: 3px;
 }
 
-.carousel__slider__text-slider .heading--oranges {
-  top: 20px;
-  left: 18px;
-  white-space: pre-wrap;
-  position: absolute;
-  font-size: 28px;
-  letter-spacing: 3px;
+.carousel-text-container__text-slider {
+  display: flex;
+  width: 300%;
+  flex-shrink: 0;
+  transition: all 1s;
 }
 
-.carousel__slider__text-slider .subheading {
-  top: 55px;
-  left: 18px;
-  position: absolute;
+.carousel-text-container__text-slider .subheading {
   text-transform: lowercase;
   font-weight: 550;
   font-size: 22px;
   letter-spacing: 2px;
 }
 
-
-.carousel__slider__text-slider .subheading--oranges {
-  top: 85px;
-  padding-right: 100px;
-  left: 18px;
-  position: absolute;
+.carousel-text-container__text-slider .subheading--oranges {
+  top:35px;
   text-transform: lowercase;
   font-weight: 550;
```

Comment:
This won't work. Font-weight for non-variable fonts has to be defined with step of a 100 units

Reply:
Changed